graphql_compiler/schema_transformation/utils.py

```diff
@@ -342,14 +344,15 @@ def check_ast_schema_is_valid(ast):
     """
     try:
         schema = build_ast_schema(ast)
-    except Exception as e:  # Can't be more specific -- see graphql/utils/build_ast_schema.py
+    except Exception as e:  # Can't be more specific -- see graphql/utilities/build_ast_schema.py
+
         raise SchemaStructureError(u"Input is not a valid schema. Message: {}".format(e))
 
-    if schema.get_mutation_type() is not None:
+    if schema.to_kwargs()["mutation"] is not None:
```

Comment:
It might be possible to avoid the `to_kwargs()` here and below on line 355 similarly to previous instances as well.